tests/modules/selenium/test_selenium_builder.py

```diff
@@ -691,9 +691,9 @@ def test_mix_syntax(self):
             content = fds.read()
 
         target_lines = [
-            "var_loc_keys=self.loc_mng.get_locator([{'name':'btn1',}])self.driver.find_element"
+            "var_loc_keys=self.loc_mng.get_locator([{'name':'btn1',}],30.0)self.driver.find_element"
```

Comment:
For me would be great don't repeat timeout sending in every get_locator call. Looks like it should be sent into LocatorsManager init. And possibly it should me non mandatory, something like that:
`def __init__ (self, locators, timeout=60):`